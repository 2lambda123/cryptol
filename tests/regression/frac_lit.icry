0b1.1     : Rational
0o1.1     : Rational
1.1       : Rational
0x1.1     : Rational
0b1.1p3   : Rational
0o1.1p3   : Rational
1.1e3     : Rational
0x1.1p3   : Rational
0b1.1p-3  : Rational
0o1.1p-3  : Rational
1.1e-3    : Rational
0x1.1p-3  : Rational

