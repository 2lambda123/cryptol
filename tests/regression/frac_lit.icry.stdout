Loading module Cryptol
(ratio 3 2)
(ratio 9 8)
(ratio 11 10)
(ratio 17 16)
(ratio 12 1)
(ratio 9 1)
(ratio 1100 1)
(ratio 17 2)
(ratio 3 16)
(ratio 9 64)
(ratio 11 10000)
(ratio 17 128)
